pydicom/tests/test_json.py

```diff
@@ -74,7 +82,11 @@ def test_json_roundtrip():
     jsonmodel = ds.to_json(bulk_data_threshold=100)
     ds2 = Dataset.from_json(jsonmodel)
 
-    assert ds2.SpecificCharacterSet == ['ISO_IR 100']
+    for e1, e2 in zip(ds, ds2):
+        if e1 != e2:
+            print(e1, e2)
+    # assert ds == ds2
```

Comment:
This shall actually be commented in instead of the debug code before - will correct this!